src/components/typography/Link/Link.tsx

```diff
@@ -15,7 +15,7 @@ const StyledLink = styled.a.attrs(() => ({
   border: none;
   font-family: inherit;
   font-weight: normal;
-  text-decoration: none;
+  text-decoration: ${getButtonColor('decoration')};
```

Comment:
I don't think retrieving text-decoration value with `getButtonColor` is the correct approach.  Also text-decoration in context of buttons doesn't make much sense to me. I personally would move text-underline in the typography part of theme.

Reply:
yeah, I was thinking whether we'll also want to have 'footprint' text button? since Link styles copy button styles, maybe we want to have 'footprint' text button... if not, I can move this elsewhere...

Reply:
so in fact, it may be better to completely separate button & link styles for clarity and consistency...

Reply:
I don't think we need "footnote" button just yet so I don't have any problem if we move the colours from Buttons into Typography, but I'd like to have that connection between those styles (so reference primary colours from buttons in typography). Then we can introduce method `getLinkStyle` (similar to `getFormStyle`) to retrieve the theme values.